jetstream/config.py

```diff
@@ -761,12 +815,11 @@ def resolve(
             raise Exception("Can't resolve an AnalysisSpec twice")
         self._resolved = True
 
+        # should experimenter.outcomes be passed into parameters merge?
```

Comment:
```suggestion
```

Not sure if I interpret this question correctly, but it's correct to pass `outcome` into `merge_parameters()` which will be an outcome that is actually used for the experiment.

Reply:
sorry, I think this is just a left over comment when I was trying to work out the flow of data and how it was being transformed by the logic. It was just to help me think. I will remove it.